pallet/src/lib.rs

```diff
@@ -104,7 +115,11 @@ pub mod pallet {
         /// Add a new item to the storage
         #[pallet::call_index(0)]
         #[pallet::weight(T::WeightInfo::add_item())]
```

Comment:
I saw the implementation is like the one below in the peak-pallet-did. Do you think we should apply the similar settings?

```
        #[pallet::weight(T::WeightInfo::add_attribute(value.len() as u32))]
```

Reply:
For this one we don't need to do this since there is a fixed maximum length to the bounded_vec. We only needed to do that for the did pallet since the vec didn't have limit. The benchmark is taking the maximum length for the calculation.